homeassistant/components/homematicip_cloud/strings.json

```diff
@@ -23,7 +23,7 @@
     "abort": {
       "unknown": "Unknown error occurred.",
```

Comment:
```suggestion
      "unknown": "[%key:common::config_flow::error::unknown%]",
```

Reply:
I didn't change that one because the parent of this 'unknown' is 'abort' and you are refering to a key from 'error'? Not sure if that is okay.